lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc

```diff
@@ -40,28 +40,35 @@ Students discover the Multiplicative Inverse Property, which tells us that multi
 
 === Launch
 
-We know that the Commutative and Associative Properties apply to multiplication...but not division. But what if there was a way to _rewrite_ division as multiplication? Then we could apply the Commutative and Associative Properties to division expressions, too! Let’s explore this idea.
+We know that the Commutative and Associative Properties apply to multiplication... but not division. But what if there was a way to _rewrite_ division as multiplication? Then we could apply the Commutative and Associative Properties to division expressions, too! Let’s explore this idea.
 
-But first, let's play a game. Draw a two-column table on the board to record response. Label the left-hand side "Starting Value," and label the right-hand side "?" (see example below).
+But first, let's play a game. I will draw a two-column table on the board.I will label the left-hand side "Starting Value," and I will label the right-hand side "?"
 
+
+@ifnotslide{
```

Comment:
Why `ifnotslide`? Use the `@table{}` directive to have this work for both lesson plans and slides!

Reply:
I did not know that!